test/fixtures/wallet-web/docker-compose-web.yml

```diff
@@ -59,6 +60,7 @@ services:
       - USE_EDV_BATCH=true
       - STATIC_ASSETS=/etc/static
       - STATIC_ASSETS_URL=
+      - CONTEXT_PROVIDER_URL=
```

Comment:
Did you miss adding file server here? and also you may need to pass file server URL as agent opts for web wallet.